package.json

```diff
@@ -28,8 +29,10 @@
   "homepage": "https://github.com/los-deferal-02/Hello-Books#readme",
   "dependencies": {
     "@babel/polyfill": "^7.4.4",
+    "bcrypt": "^3.0.6",
```

Comment:
Bro, sorry. Bcrypt doesn't work on my PC. I have issues with its dependencies. Could you please consider using `bcryptjs` npm package instead?

Reply:
Okay bro. I will.